app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -103,12 +102,12 @@
     <string name="semi_column">:</string>
 
     <!-- New -->
-    <string name="no_watch">No Watch</string>
-    <string name="set_time">Set Time</string>
-    <string name="set_reminders">Set Reminders from Google Calender</string>
-    <string name="take_photo">Take a photo</string>
-    <string name="toggle_flashlight">Toggle Flashlight</string>
-    <string name="start_voice_assistant">Start Voice Assist</string>
-    <string name="emergency_actions">Emergency Actions:</string>
-    <string name="make_phonecall">Make a phone call</string>
+    <string name="no_watch">未添加手表</string>
+    <string name="set_time">自动设置事件</string>
+    <string name="set_reminders">自动从Google Calender添加待办</string>
+    <string name="take_photo">照相</string>
+    <string name="toggle_flashlight">启动闪光灯</string>
```

Comment:
Really nit-picking here, but ChatGPT translates this to "Turn on the flashlight". A more precise description would be "Toggle Flashlight", since repeatedly running this action would turn the flash on and off.

Reply:
If repeatedly running this action would turn the flash on and off, The translation should not be“打开闪光灯” but “切换闪光灯状态” instead, That's my problem, sorry